modules/Kconfig.lz4

```
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
```

when moving this file, you must add:
```suggestion

config ZEPHYR_LZ4_MODULE
        bool

```